### PR TITLE
Describe environment variables used to run the job

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related Issues and Dependencies

Related to #96 

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

List the environment variables used to run and test `prescriptions-refresh-job`.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,3 +2,30 @@ Thoth's prescriptions-refresh-job
 ---------------------------------
 
 A job that keeps `Thoth's prescriptions <https://github.com/thoth-station/prescriptions>`__ up to date.
+
+Environment variables available to run and test the job:
+========================================================
+- APP_SCRIPT
```
Comment (pacospace) [inline README.rst:8]: what about some description of them?

**Review by pacospace — COMMENTED** [review]
very nice! @mayaCostantini ! Thanks! lgtm!

for the remaining ones, let's wait for @fridex 🍻

**Review by fridex — APPROVED** [review]
lgtm, thanks! 💯 👍🏻

**Pull request merged**